14.9/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# vim:set ft=dockerfile:
+
+FROM cimg/base:2020.06
```

Comment:
Should this be 08?

Reply:
No, there weren't any major changes in the past few monthly snapshots, so I've been keeping it as June to keep the cache hit rate up. The next planned bump will be for next month, September,  because that's when we're getting Ubuntu 20.04.